test/gtest/uct/test_md.cc

```diff
@@ -943,7 +944,8 @@ UCS_TEST_SKIP_COND_P(test_md, rkey_compare, !check_caps(UCT_MD_FLAG_REG))
     uct_mem_h memh1, memh2;
     int result1, result2;
 
-    if (GetParam().md_name == "cuda_ipc") {
+    if (GetParam().md_name == "cuda_ipc" ||
+        GetParam().md_name == "rocm_ipc") {
```

Comment:
1. maybe add helper function to eliminate duplication?
2. use it as part of skip condition in the UCS_TEST_SKIP_COND_P macro
3. code style - add () around each == expression

Reply:
Added a new helper function (check_md_name) to the skip condition in the UCS_TEST_SKIP_COND_P macro.